src/devices_models/devices/common/range_constraint.jl

```diff
@@ -22,28 +32,17 @@ where limits in constraint_infos.
 `` x = limits^{max}, \text{ for } |limits^{max} - limits^{min}| < \varepsilon ``
 
 `` limits^{min} \leq x \leq limits^{max}, \text{ otherwise } ``
-
-# Arguments
-* psi_container::PSIContainer : the psi_container model built in PowerSimulations
-* constraint_infos::Vector{DeviceRangeConstraintInfo} : contains names and vector of min/max
-* cons_name::Symbol : name of the constraint
-* var_name::Symbol : the name of the continuous variable
 """
-function device_range(
-    psi_container::PSIContainer,
-    constraint_infos::Vector{DeviceRangeConstraintInfo},
-    cons_name::Symbol,
-    var_name::Symbol,
-)
+function device_range(psi_container::PSIContainer, inputs::RangeConstraintInputs)
     time_steps = model_time_steps(psi_container)
-    variable = get_variable(psi_container, var_name)
-    ub_name = middle_rename(cons_name, PSI_NAME_DELIMITER, "ub")
-    lb_name = middle_rename(cons_name, PSI_NAME_DELIMITER, "lb")
-    names = (get_name(x) for x in constraint_infos)
+    variable = get_variable(psi_container, inputs.variable_name)
```

Comment:
we should use getter functions here

Reply:
RangeConstraintInputsInternal is a simple struct with no abstract type.  Field access makes more sense to me.  Can you elaborate?

Reply:
I am trying to gauge if we should always use getter functions for structs to simplify maintenance or fields. Using getter functions made it easier for me to change the contents and structure psi_container and stage in the past.